code/modules/mob/living/living.dm

```diff
@@ -386,6 +389,10 @@
 			return
 
 	if(ismovableatom(A))
+		if(isxeno(src))
+			var/mob/living/carbon/xenomorph/meanie = src
+			if(meanie.xeno_caste.caste_flags & CASTE_CANT_PUSH && (istype(A,/mob/living/carbon/human) || istype(A, /mob/living/carbon/xenomorph)) && (a_intent == INTENT_HARM || a_intent == INTENT_DISARM || a_intent == INTENT_GRAB))
```

Comment:
should have a parent more unifying these two
so from
`(istype(A,/mob/living/carbon/human) || istype(A, /mob/living/carbon/xenomorph))`
to
`((istype(A,/mob/living/carbon/human) || istype(A, /mob/living/carbon/xenomorph)))`

Reply:
done